integr8ly/installation
# update booster catalog tag

Update to latest launcher booster catalog

Verification steps:

1. Run WT2
2. When you select the Fuse runtime, make sure its listed as `Fuse 7.4` in the dropdown.

Upgrade:

1. Install 1.4.1
2. Run the upgrade playbook
3. Check the `launcher` configmap in the launcher namespace. The key `launcher.backend.catalog.git.ref` needs to have a value of `integreatly-1.5.0`.
